src/functions-and-arrays.js

```diff
@@ -1,19 +1,103 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(a,b){
+  if(a > b){
+    return a;
+  } else if (a < b){
+    return b;
+  } else {
+    return a;
+  }
+}
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(arr) {
+  if(arr.length == 0){
+    return null;
+  } else if(arr.length == 1) {
+    return arr[0];
+  } else {
+    const lengthArr = arr.map(item => item.length);
+    const longestWord = arr[lengthArr.indexOf(Math.max(...lengthArr))];
+    return longestWord;
+  }
+}
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(arr){
+  let sum1 = 0;
+
+  for(let i=0; i < arr.length; i++){
+    sum1 += arr[i];
+  }
+
+  return sum1;
+}
+
+const mixedArr = [6, 12, 'miami', 1, true, 'barca', '200', 'lisboa', 8, 10];
+
+function sum(mixArr){
+  let sum2 = 0;
+
+  for(let j=0; j < mixArr.length; j++) {    
+    if(typeof mixArr[j] == "number") {
```

Comment:
You can actually make and or statement here: 
typeof mixArr[j] == "number" || typeof mixArr[j] == "boolean"

since a number + a boolean results in: 5 + true === 6, and 5 + false === 5